### ISSUE TITLE
Sequential test suits

### ISSUE THREAD

**Issue by PaulNBarron** [tcunit/ExampleProjects#1]
Hi,
Is it possible to run the test suits sequentially? For example if I have TestSuite1_FB and TestSuite2_FB is it possible to wait until TestSuite1_FB has finished executing before starting TestSuite2_FB?
Thanks

**Comment by sagatowski** [comment]
Duplicate of https://github.com/tcunit/TcUnit/issues/97.
